test/test_particle_hole.py

```diff
@@ -16,7 +16,7 @@
 
```

Comment:
I think we can remove both, this test file as well as `qiskit_nature.particle_hole`. A PR is on the way by @pbark anyways, correct?